scripts/development/start-dev.sh

```diff
@@ -1,13 +1,105 @@
- #!/bin/bash
+#!/bin/bash
+# export COMPOSE_HTTP_TIMEOUT=300
+# sh scripts/development/start-dev.sh
 
-if [ ! -f .env ]; then
-    cp .env_example .env
-fi
 
-echo "Building, Starting and Provisioning TruBudget for Development"
+Help()
+{
+    # Display Help
+    echo
+    echo "Help"
+    echo "This script setups a TruBudget Development instance of your choice."
+    echo "Frontend, API, Excel-Export, Email-Notification, Storage-Service are always hot reloaded on save."
+    echo "The Multichain data is NOT persisted and will be lost after a restart."
+    echo "The environmental variables are set automatically."
+    echo
+    echo "Hint: Make sure you are in the root directory of TruBudget."
+    echo
+    echo "Syntax: sh scripts/development/start-dev.sh [options]"
+    echo "Example: sh scripts/development/start-dev.sh --full --build"
+    echo
+    echo "options:"
+    echo "      --slim                  Starts a TruBudget instance with master-node, master-api, provisioning and frontend."
+    echo "      --full                  Starts a TruBudget instance with master-node, emaildb, minio, master-api, email-service,"
+    echo "                              provisioning, excel-export-service, storage and frontend."
+    echo "      --build                 Force docker-compose built"
+    echo "-h  | --help                  Print the help section"
+    echo
```

Comment:
What's about this script feedback?:
without any option only blockchain/api/frontend shall start
--provision (start provisioning service)
--enable-service export (source additional env vars & start export service)
--enable-service email (source additional env vars & start email service)
--enable-service storage (source additional env vars & start stoprage service and minio)
--full as above
--build as above
--debug [service] (enable hotreloading for the service specified)
-h --help as above

Reply:
for tgis feedback, I created a follow up issue: #904